<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DHLR</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>DHLR.ico</normaloff>DHLR.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>750</width>
      <height>430</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btnQuery">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>100</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>360</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maxLength">
     <number>15</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>310</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string> MSISDN (eg. 861xxxxxxxxxx)
 -- or --
 IMSI (eg. 46001xxxxxxxxxx)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLocUpd">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>100</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>位置更新</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn4to3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>50</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>4G -&gt; 3G</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn3to2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>70</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>3G -&gt; 2G</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn3to4">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>50</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>3G -&gt; 4G</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn2to3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>70</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>2G -&gt; 3G</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnQueryOther">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>100</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>查询位置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCFx">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>100</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>呼叫转移</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRest">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>70</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>复机</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStop">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>50</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>停机</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>inputBox</tabstop>
  <tabstop>btnQuery</tabstop>
  <tabstop>btnLocUpd</tabstop>
  <tabstop>btnQueryOther</tabstop>
  <tabstop>btnCFx</tabstop>
  <tabstop>btn4to3</tabstop>
  <tabstop>btn3to4</tabstop>
  <tabstop>btn3to2</tabstop>
  <tabstop>btn2to3</tabstop>
  <tabstop>btnStop</tabstop>
  <tabstop>btnRest</tabstop>
  <tabstop>textBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
